CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ IMPROVEMENTS:
 
 * Add missing documentations for Federated repo resources [GH-304]
 * Add additional repo types for Federated repo resources [GH-304]
+* Added following smart remote repo attributes for remote repository resources [GH-50].
```

Comment:
Please create a new section for 2.14.0

Reply:
I don't think TF actually replaces `(Unreleased)` with actual dates. So please also add today's date.

Reply:
The GH reference is for the PR, not the original issue.